### PR TITLE
Working With Animations - SpriteAnimator Update

### PR DESCRIPTION
**Pull request by padesso**
### Summary

This PR updates the Working with Animations section of the v3 docs to comply with the updated workflow and code changes for using the SpriteAnimator to do animations.

### Article Checklist

- [x ] A table-of-contents marker is present below the article summary, above the first chapter.
- [x ] Spell checking software doesn't complain.
- [x ] All images are part of the PR, or already in the repository.
- [x ] All links are verified to work.
- [x ] Language used is (overall) clean, precise and to the point.

### Technical Checklist

- [x ] Provide a bulleted list of changes that are introduced by this PR.
- [x ] Screenshots of before / after for style and layout related changes.

### CODE REVIEW

File: pages/v3/Working-With-Animations.md [hunk 1]
```diff
@@ -12,30 +12,37 @@ Very few games are made that don't need animations. Duality supports animations
...
+![](../img/Animation/Animation2_5.png)
 
 Create a new GameObject by right-clicking in the scene view and give it a name: Billy (I call it that)
-In Billy, add a new AnimSpriteRenderer Component.
+In Billy, add a new SpriteAnimator (New -> Graphics -> SpriteAnimator).  This will automatically create a Transform and SpriteRenderer in your GameObject.
 
 ![](../img/Animation/Animation3.png)
 
-You will Duality's logo appearing. That's ok. Drag Billy's Pixmap over AnimSpriteRenderer and you'll see Billy in your scene.
+You will Duality's logo appearing. That's ok. In the SpriteRenderer, drag Billy's Pixmap from the Project View window over the SharedMaterial and update the width and height of the SpriteRenderer to match the size of the spritesheet to a height of 205 and a width of 231.  Set the AnimDuration to 1.0, the FirstFrame to 0 and the FrameCount to 4 (The walk animation is 4 frames long).
```
Comment (SirePi) [inline pages/v3/Working-With-Animations.md]: typo: You will _see_ Duality...

in addition, there's something wrong setting the SpriteRenderer's size to 231x205; that's the spritesheet's original size, but it contains a non-square grid of 3x2 sprites, which means that the ratio is off and it is even scaled up. The "correct" values to have the renderer display a 1:1 version of the single sprite should be 

| -35.0 | -51.25 |
| ---:| ---:|
| 77.0 | 102.5 |

(size / grid, then /2 to have the single sprite's offset)

**Review by SirePi — CHANGES_REQUESTED** [review]
1 typo, 1 passage maybe not giving the expected results

File: pages/v3/Working-With-Animations.md (hunk 1 above)
Comment (SirePi) [reply pages/v3/Working-With-Animations.md]: For the scaled up part, maybe it's enough to just add a remark about it

Comment (padesso) [reply pages/v3/Working-With-Animations.md]: Thanks for the feedback - updating shortly.

**Review by SirePi — APPROVED** [review]
Looks good to me 👍

**Review by ilexp — APPROVED** [review]

**Pull request merged**